src/test/java/io/getunleash/example/UnleashUsageTest.java

```diff
@@ -15,7 +15,7 @@ public void wire() {
                 new UnleashConfig.Builder()
                         .appName("test")
                         .instanceId("my-hostname:6517")
-                        .synchronousFetchOnInitialisation(true)
+                        .synchronousFetchOnInitialisation(false)
```

Comment:
There's no localhost:4242 set up in this test, so this test would fail if we fetch synchronously